tests/smartswitch/common/device_utils_dpu.py

```diff
@@ -85,10 +85,9 @@ def dpu_power_on(duthost, platform_api_conn):
         Returns True or False based on all DPUs powered on or not
     """
 
-    num_modules = num_dpu_modules(platform_api_conn)
```

Comment:
do we need to add num_dpu_modules to the function?

Reply:
Yes, it was given as comment in last PR.